tap_google_sheets/sync.py

```diff
@@ -62,6 +62,7 @@ def sync(client, config, catalog, state):
 
         # sync file metadata
         elif stream_name == "file_metadata":
+            stream_obj.params['supportsAllDrives'] = client.supports_all_drives
```

Comment:
Doesn't the `stream_obj` use the `client` to make the request to Google? Why does the `client` need to store `supports_all_drives`? This feels backwards and we are blurring the responsibilities of the objects here

Reply:
@luandy64, updated code as per your suggestion and removed supports_all_drives from the client.